### DISCUSSION TITLE
Make sure site works on mobile.

### DISCUSSION THREAD

**Discussion by davidkinetisense** [110yards/fantasy#94] [Ideas]
Right now the scores on work on desktop.  Please consider a mobile option.

**Comment by mdryden** [comment]
The scores work on mobile as well, I use mobile almost exclusively.

I think the issue you are having is related to the way the scoring is calculated; it results in burst of requests being sent to the back end at one time, and those requests often fail.

I'm going to simplify things for next year, it should help.
